tests/integ/test_workflow.py

```diff
@@ -401,3 +407,101 @@ def test_conditional_pytorch_training_model_registration(
             pipeline.delete()
         except Exception:
             pass
+
+
+def test_training_job_with_debugger(
+    sagemaker_session,
+    pipeline_name,
+    role,
+):
+    instance_count = ParameterInteger(name="InstanceCount", default_value=1)
+    instance_type = ParameterString(name="InstanceType", default_value="ml.m5.xlarge")
+
+    rules = [
+        Rule.sagemaker(rule_configs.vanishing_gradient()),
+        Rule.sagemaker(base_config=rule_configs.all_zero(), rule_parameters={"tensor_regex": ".*"}),
+        Rule.sagemaker(rule_configs.loss_not_decreasing()),
+    ]
+    debugger_hook_config = DebuggerHookConfig(
+        s3_output_path=os.path.join(
+            "s3://", sagemaker_session.default_bucket(), str(uuid.uuid4()), "tensors"
+        )
+    )
```

Comment:
please avoid use of `os.path.join` for s3 paths as `os.path` delimiter varies by platform, whereas s3 is always `/`:

something like this:
```suggestion
        s3_output_path=f"s3://{sagemaker_session.default_bucket()}/{uuid.uuid4()}/tensors"
```

Reply:
Sure